onnxruntime/core/graph/schema_registry.cc

```diff
@@ -154,6 +154,29 @@ void SchemaRegistryManager::RegisterRegistry(std::shared_ptr<IOnnxRuntimeOpSchem
   registries.push_front(registry);
 }
 
+static int GetActualLastSupportedOpSet(
```

Comment:
How does this relate to the "DomainToVersionMap" in graph and model ?

Reply:
Hmm, I fear my latest iteration last night (in an attempt to pacify the test code which calls this function to indirectly figure out what opset it should use for dynamically generated models by default) will break app registered schemas :/, and we do want to support that.